battleship.py

```diff
@@ -77,7 +77,15 @@ def emptyGrid(rows, cols):
 Returns: 2D list of ints
 '''
 def createShip():
-    return
+    row_centre=random.randint(1,8)
+    col_centre=random.randint(1,8)
+    orientation=random.randint(0,1)
+    ship_placement=[[],[],[]]

```

Comment:
@priyushasunkara What does 84th line will do in this function?

Reply:
it checks if the orientation of the ship is vertical(0) or horizontal(1)

Reply:
Thank you for the update in the code. Good job!!!

Reply:
Sir, Can I merge the code in main?